app/Models/BoardCat.php

```diff
@@ -43,11 +43,11 @@ class BoardCat extends Model
 
     protected $guarded = [];
 
-    public function threads(){
+    public function threads() {
         return $this->hasMany('App\Models\BoardThread', 'cat_id', 'id');
     }
 
-    public function last_user(){
+    public function last_user() {
```

Comment:
__[code_sniffer]__
* Method name "BoardCat::last_user" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->